### PR TITLE
mitigate ws and braces vulnerabilities

### PR DESCRIPTION
**Pull request by ben-polinsky**
Mitigates [GHSA-3h5v-q93c-6h6q](https://github.com/advisories/GHSA-3h5v-q93c-6h6q), [GHSA-grv7-fg5c-xmjg](https://github.com/advisories/GHSA-grv7-fg5c-xmjg) 

Unfortunately, pnpm global overrides seems to be the only way to go for braces which is a deep transient dependency off react-scripts.

### CODE REVIEW

File: rush.json [hunk 1]
```diff
@@ -2,9 +2,6 @@
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
   "rushVersion": "5.112.2",
   "pnpmVersion": "7.32.2",
-  "pnpmOptions": {
-    "useWorkspaces": true
```
Comment (aruniverse) [inline rush.json:6]: any reason for removing this?

**Review by aruniverse — COMMENTED** [review]
`rush update --full` didnt work?

File: rush.json (hunk 1 above)
Comment (ben-polinsky) [reply rush.json:6]: Moved to `pnpm-config.json` as requested by rush

**Comment by ben-polinsky** [comment]
> `rush update --full` didnt work?

Nope couldn't get `braces` updated...

**Comment by aruniverse** [comment]
> > `rush update --full` didnt work?
> 
> Nope couldn't get `braces` updated...

Do we still need to add the `ws` dep in the test apps?

**Comment by ben-polinsky** [comment]
> > > `rush update --full` didnt work?
> > 
> > 
> > Nope couldn't get `braces` updated...
> 
> Do we still need to add the `ws` dep in the test apps?

nope - makes sense

**Comment by ben-polinsky** [comment]
@jasdom since I am running into issues which cross over into #319, maybe you can pick the changes relating to the failed audit I was attempting to resolve

**Comment by jasdom** [comment]
> @jasdom since I am running into issues which cross over into #319, maybe you can pick my changes here relating to the failed audit I was attempting to resolve

Sure, I was working on these exact changes to resolve the vulnerabilities myself so I will include them in my PR.

**Pull request closed without merging**